lib/pod-names.js

```diff
@@ -0,0 +1,32 @@
+var Plugin = require('broccoli-plugin');
+var walkSync = require('walk-sync');
+var fs = require('fs');
+var path = require('path');
+var componentNames = require('./component-names.js');
+
+module.exports = PodNames;
+
+PodNames.prototype = Object.create(Plugin.prototype);
+PodNames.prototype.constructor = PodNames;
+function PodNames(inputNode, options) {
+  options = options || {};
+  Plugin.call(this, [inputNode], {
```

Comment:
persistentOutput is a mode one can put a broccoli plugin in, this allows it to take over management of its output directory and what enabled merge-trees + persistent-filter + funnel to get a 10x speed boost.